railsrc

```diff
@@ -0,0 +1,2 @@
+--database=postgresql
+--css=tailwind
```

Comment:
How do we feel about this?

Reply:
Is this what we use to initialize most new apps these days? I haven't done `rails new` in a few years 😅

Reply:
This causes Rails to run `rails css:install:tailwind` after making the app. We can do that ourselves if we want Tailwind, not sure why it needs to be part of `rails new`.

Reply:
It looks like all the CSS frameworks depend on `css:install`, which might bring in [good stuff](https://github.com/rails/cssbundling-rails/blob/main/lib/install/install.rb). Is that the only way to get `css:install` to run?

Reply:
On the Ignite team, we use Tailwind by default. At least we've used it on the last few projects I was involved in. I would like to hear from @kvnkwon  and @vendelal (both out sick today) if they feel like we should continue to use it going forward.

However Ignite rarely starts a new project in Rails so our opinion may be irrelevant here.

Reply:
I should point at that right now, [suspenders generates new apps with postcss](https://github.com/thoughtbot/suspenders/blob/55cb5c246fe53aea4cf53436c7c208b40ad13e85/lib/suspenders/generators/app_generator.rb#L34-L35).

It sounds like there's no disagreement about using PostgreSQL as our preferred database, but it does sound like there's a disagreement in how we prefer to style applications.

Reply:
I think because there's disagreement it points to there not being alignment on a standard. In the css instance, personally I feel tailwind or other frameworks should be opt-in. That's because I'm not really a fan, and I know there are at least a handful of others that would rather not work in tailwind given the chance.

Reply:
I ended up removing `--css=tailwind` based on this discussion. We can consider addressing how we want to handle styles in a [new version of Suspenders](https://github.com/thoughtbot/suspenders/pull/1135) I'm exploring.